README.md

```diff
@@ -2,9 +2,13 @@
 De documentatie is nog volop in ontwikkeling
 ```
 
-# Kennisbank
+## OSPO-NL Kennisbank
 
-Deze documentatie is gepubliceerd op [ospo-nl.github.io/kennisbank](https://ospo-nl.github.io/kennisbank/).
+De OSPO-NL kennisbank is een verzameling van kennis en best practices voor het opzetten en uitvoeren van OSPO's (of OSPOs) bij organisaties in Nederland.
+
+De OSPO-NL kennisbank is gepubliceerd op [ospo-nl.github.io/kennisbank](https://ospo-nl.github.io/kennisbank/).
+
+![image](https://github.com/ospo-nl/kennisbank/blob/feature/update-readme.md/docs/assets/images/Screenshot%20kennisbank%20OSPO-nl.png)
```

Comment:
Relatieve verwijzing is voldoende ... en zorgt dat het beter en direct vanuit de gepubliceerde sources wordt geserveerd in de browser. Bovendien heb je nu zelfs de branch naam opgenomen en zal het plaatje niet meer zichtbaar zijn als deze branch wordt opgeruimd 😬  Beter om het zo te doen:

```markdown
![Kennisbank Screenshot](assets/images/Screenshot%20kennisbank%20OSPO-nl.png)
```

(ik ben zelf geen fan van spaties in bestandsnamen ... omdat deze moeilijker URL encoded worden. Dat betekent namelijk `%20` in de naam en dat vind ik minder mooi dan een `-` of een `_`. Niet noodzakelijk om te veranderen ... wel mooier - vind ik 😊 )

Reply:
Een relatieve verwijzing is idd een goed idee. Dat had ik eigenlijk zelf al moeten bedenken  😬